src/main/java/bio/terra/service/job/StairwayLoggingHooks.java

```diff
@@ -16,31 +17,43 @@
         "timestamp: {}";
     private static final Logger logger = LoggerFactory.getLogger(StairwayHook.class);
 
+    private PerformanceLogger performanceLogger;
+
+    public StairwayLoggingHooks(PerformanceLogger performanceLogger) {
+        this.performanceLogger = performanceLogger;
+    }
+
     @Override
     public HookAction startFlight(FlightContext context) {
         logger.info(FlightLogFormat, "startFlight", context.getFlightClassName(),
             context.getFlightId(), Instant.now().atZone(ZoneId.of("Z")).format(DateTimeFormatter.ISO_INSTANT));
+        performanceLogger.timerStart("stairwayFlight" + context.getFlightId());
```

Comment:
Timers won't work for flights. Flight can migrate between threads and DRmanagers.

Reply:
Updated to just log with a timestamp instead of trying to use a timer. Even if it worked some of the time, we'd still have to handle the cross DR Manager case, so this was not useful. Thanks for the catch.